src/osha/oira/content/browser/survey_publication_count.py

```diff
@@ -0,0 +1,60 @@
+from io import StringIO
+from plone import api
+from Products.Five import BrowserView
+
+import csv
+
+
+fieldnames = ["country", "path", "count"]
```

Comment:
Later you set this as a view attribute.
This should become a class attribute of the `SurveyPublicationCount` view.